.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "typescript.tsdk": "node_modules\\typescript\\lib"
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Please do not include this change since it's the local vscode environment settings.